AdminUi/src/AdminUi/appsettings.json

```diff
@@ -1,5 +1,6 @@
 {
     "AllowedHosts": "*",
+    "Urls": "http://*:5000",
```

Comment:
I remember discussing this property with you, and you were unsure whether it was actually doing something.
Did you get a chance to validate that?

Reply:
Yes! It affects the port where the API is served in the docker container.